Cave Flier/Assets/Scripts/playerMovement.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+using UnityEngine;
+using System;
+using System.Globalization;
+
+
+public class playerMovement : MonoBehaviour
+{
+    public float speed = 6;
+
+	// Use this for initialization, called on first frame the object is active
+	void Start ()
```

Comment:
incorrect spacing.  the function call and comment are further indented than the opening brace.